### PR TITLE
RLM-204 Move Kilo Leapfrog Jobs to Periodic Only

### PR DESCRIPTION
**Pull request by antonym**
Adds Kilo Leapfrog PRs to the exclude list for MultiAIO and AIO:

Issue: [RLM-204](https://rpc-openstack.atlassian.net/browse/RLM-204)

### CODE REVIEW

File: rpc_jobs/multi_node_aio.yml [hunk 1]
```diff
@@ -101,7 +101,11 @@
       # Leapfrog upgrades are only run for kilo --> newton141
       # as the upgrade method is not supported for any
       # other target series.
-      # the kilo --> newton141 job is triggered by PRs to either branch.
+      # the kilo --> newton141 job is only ran as a periodic job until
+      # it's more reliable and can be tested as a PR
+      - series: kilo
+        action: leapfrogupgrade
+        trigger: pr
```
Comment (mattt416) [inline rpc_jobs/multi_node_aio.yml]: How about:

```
- action: leapfrogupgrade
   trigger: pr
```

Comment (antonym) [reply rpc_jobs/multi_node_aio.yml]: K, updated with the two lines as it sounds like using three entries in an exclude doesn't work.

**Comment by mattt416** [comment]
We'll need to delete jobs after this one merges.  Any issues w/ that @hughsaunders and @git-harry ?

**Comment by hughsaunders** [comment]
project_builds failed as PR is from a fork.

**Pull request merged**